### DISCUSSION TITLE
Mod Updater

### DISCUSSION THREAD

**Discussion by HoodieRocks** [modrinth/.github#24] [Feature requests]
**Is your feature request related to a problem? Please describe.**
It's difficult to find if a mod is updated or if it's out of date

**Describe the solution you'd like**
A (JSON?) based update system. Drag and drop a JSON file containing your mods and their versions. Modrinth will look through the JSON file and find any updates to mods. If an update is available it will give you a list of outdated mods and links to those mods.

**Describe alternatives you've considered**
Possibly a page where you input the name of the mods along with their versions and it will give you a list of mods that are outdated

**Comment by Redblueflame** [comment]
Hi!
This feature is already partly implemented with the follow system. Each time that a mod you follow is updated, you get immediately notified.

But the json idea is quite good, even if it relies on third party services to generate the json file in the first place.

In all cases, I'll keep that issue open for when modpacks are implemented.

**Comment by triphora** [comment]
I feel as if this is best left to external providers such as [Mod Manager](https://modrinth.com/mod/modmanager) or [Modget](https://modrinth.com/mod/modget); there are already a million implementations of mod updaters that I don't think introducing an official competitor is a good idea.
